helm-chart/splunk-connect-for-kubernetes/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: splunk-connect-for-kubernetes
-version: 1.0.0
+version: 1.1.0
```

Comment:
Please do not change the versions in this PR. We will bump the versions once we are ready to release a new version.